advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,461 @@
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/CollectionResponse[OperationEntity]"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/{recomendationId}/suppressions/{name}": {
+			"put": {
+				"tags": ["Suppression"],
+				"operationId": "Suppression_Suppressions",
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				},
+				{
+					"name": "recomendationId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"name": "api-version",
+					"in": "query",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"name": "name",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object"
+						}
+					}
+				},
+				"deprecated": false
+			},
+			"delete": {
+				"tags": ["Suppression"],
+				"operationId": "Suppression_Delete",
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"parameters": [{
+					"name": "recomendationId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"name": "name",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				},
+				{
+					"name": "api-version",
+					"in": "query",
+					"required": true,
+					"type": "string"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/microsoft.Advisor/suppressions": {
+			"get": {
+				"tags": ["Suppression"],
+				"operationId": "Suppression_GET",
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				},
+				{
+					"name": "api-version",
+					"in": "query",
+					"required": true,
+					"type": "string"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		}
+	},
+	"definitions": {
+		"ResourceRecommendationDetail": {
+			"type": "object",
+			"properties": {
+				"actions": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/ResourceRecomendationAction"
+					}
+				},
+				"description": {
+					"type": "string"
+				},
+				"label": {
+					"type": "string"
+				},
+				"learnMoreLink": {
+					"type": "string"
+				},
+				"category": {
+					"enum": ["HighAvailability",
+					"Security",
+					"Performance",
+					"Cost"],
+					"type": "string"
+				},
+				"id": {
+					"type": "string"
+				},
+				"impact": {
```

Comment:
`x-ms-enum` Refer https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-enum and apply as necessary to model SDK correctly.